heimdall2.code-workspace

```diff
@@ -25,6 +25,35 @@
       "**/dist": true,
       "**/samples": true,
       "**/parse_testbed": true
-    }
+    },
+    "yaml.customTags": [
+      "!And",
+      "!And sequence",
+      "!If",
+      "!If sequence",
+      "!Not",
+      "!Not sequence",
+      "!Equals",
+      "!Equals sequence",
+      "!Or",
+      "!Or sequence",
+      "!FindInMap",
+      "!FindInMap sequence",
+      "!Base64",
+      "!Join",
+      "!Join sequence",
+      "!Cidr",
+      "!Ref",
+      "!Sub",
+      "!Sub sequence",
+      "!GetAtt",
+      "!GetAZs",
+      "!ImportValue",
+      "!ImportValue sequence",
+      "!Select",
+      "!Select sequence",
+      "!Split",
+      "!Split sequence"
+    ]
   }
```

Comment:
Does this belong in the shared workspace?

Reply:
This looks like something that was automatically added by a plugin on George's computer (yaml extension pack) but isn't necessary for this change, removed now.